x/gamm/client/cli/tx.go

```diff
@@ -290,6 +291,17 @@ Ex) 2,4,1,5 -> [(Balancer 2, CL 4), (Balancer 1, CL 5)]
 	return cmd
 }
 
+func NewMigrateSharesToFullRangeConcentratedPositionCmd() *cobra.Command {
```

Comment:
I think we can make it return `*osmocli.TxCliDesc` and use `osmocli.AddTxCmd` instead of using BuildCommandCustomFn since we are not using any custom flags here

Reply:
Agreed on this